### ISSUE TITLE
pip install leads to "VSPAero solver not found"

### ISSUE THREAD

**Issue by jeffallen** [OpenVSP/OpenVSP#243]
After following the instructions in /opt/OpenVSP/python to do an install into a Conda environment, we found that the vspaero, vspviewer and vsploads binaries are not present in the $HOME/.conda/envs/vsppytools/lib/python3.10/site-packages/openvsp directory.

The binaries were made by CMake correctly, and ended up in the RPM, and the RPM put them in /opt/OpenVSP/python/openvsp/openvsp. The place they get lost is when `pip install -r requirements.txt` uses `setup.py` to figure out what to install. So it seems like the right magic to tell `setup.py` about non-Python elements of the install is missing.

As a workaround, we were able to fix it like this:
```
cp /opt/OpenVSP/python/openvsp/openvsp/vsp* $HOME/.conda/envs/vsppytools/lib/python3.10/site-packages/openvsp
```

cc @mumbricht

**Comment by ramcdona** [comment]
Please try adding these three lines to the MANIFEST.in file in python/openvsp/openvsp before the package installation.

include openvsp/vspaero*
include openvsp/vsploads*
include openvsp/vspviewer*

Theoretically, that should cause them to be copied along with all the *.py files.

The * is there to hopefully work on both Linux/MacOS (where it should match with nothing) and on Windows (where it should match with .exe).

**Comment by ramcdona** [comment]
This should be addressed in 3.32.0.
